buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/BuildForkPointDistribution.java

```diff
@@ -123,27 +120,22 @@ private void tryBuildDistribution(String commit) throws Exception {
     }
 
     private Object[] getBuildCommands(String commit, String baseVersion) {
-        String[] commands = new String[]{
+        BuildCacheConfiguration buildCacheConf = ((GradleInternal) getProject().getGradle()).getSettings().getBuildCache();
+        HttpBuildCache remoteCache = (HttpBuildCache) buildCacheConf.getRemote();
+        boolean remoteCacheEnabled = remoteCache == null ? false : remoteCache.isEnabled();
+
+        return new String[]{
             "./gradlew",
-            "install",
+            "clean",
+            ":install",
             "-Pgradle_installPath=" + getDestinationFile("gradle-" + baseVersion + "-commit-" + commit),
             ":toolingApi:installToolingApiShadedJar",
-            "-PtoolingApiShadedJarInstallPath=" + getDestinationFile("gradle-tooling-api-" + baseVersion + "-commit-" + commit + ".jar")
+            "-PtoolingApiShadedJarInstallPath=" + getDestinationFile("gradle-tooling-api-" + baseVersion + "-commit-" + commit + ".jar"),
+            "-Dorg.gradle.caching=" + (remoteCache == null ? false : remoteCache.isEnabled()),
```

Comment:
Caching should be enabled when caching for the build running this task is enabled.